strongbox-storage/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexerTest.java

```diff
@@ -118,11 +120,11 @@ public void testIndex() throws Exception
         for (SearchResult result : results)
         {
             artifactInfos.add(new ArtifactInfo(result.getRepositoryId(),
-                                               result.getGroupId(),
-                                               result.getArtifactId(),
-                                               result.getVersion(),
-                                               result.getClassifier(),
-                                               result.getExtension()));
+                    ((MavenArtifactCoordinates) result.getArtifactCoordinates()).getGroupId(),
```

Comment:
Wouldn't it be better to just define a casted variable here, instead all this casting?